openvpn_api/vpn.py

```diff
@@ -113,7 +112,27 @@ def send_command(self, cmd) -> Optional[str]:
         logger.debug("Cmd response: %r", resp)
         return resp
 
-    # Interface commands and parsing
+
+class VPN:
+    def __init__(self, connection: Connection):
+        self.connection = connection
+        self.clear_cache()  # Initialise release info and daemon state caches
+
+    @staticmethod
+    def initialize(
+        host: Optional[str] = None,
+        port: Optional[int] = None,
+        *,
+        socket: Optional[str] = None,
+        auto_connect: bool = True,
+    ):
+        """Create Connection & VPN objects in one shot.
+        """
+        connection = Connection(host, port, socket)
+        vpn = VPN(connection=connection)
+        if auto_connect:
+            connection.connect()
+        return connection, vpn
```

Comment:
Why it's weird? I saw this pattern already in some libraries (I don't remember which ones). But an module-level method instead of class static method would be good also.